src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,53 @@
-import React from 'react';
+import { useState } from 'react';
+
+import classNames from 'classnames';
+
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+import { MovieCard } from '../MovieCard';
+
+import { getMovie } from '../../api';
+import { normalizeMovieData } from './helper';
+
+import { Movie } from '../../types/Movie';
+
+type FindMovieProps = {
+  onAdd: (movie: Movie) => void;
+};
+
+export const FindMovie = ({ onAdd }: FindMovieProps) => {
+  const [movieInput, setMovieInput] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [movieError, setMovieError] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  const handleSubmit = (e: React.SyntheticEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setLoading(true);
+
+    getMovie(movieInput)
+      .then(data => {
+        if (data.Response === 'True') {
+          setMovie(normalizeMovieData(data));
+        } else {
+          setMovieError(true);
+        }
+      })
+      .finally(() => setLoading(false));
+  };
+
+  const handleAdd = () => {
```

Comment:
I would name this **addMovie** or **addMovieHandler**

Reply:
`handleAddMovie` :)